### DISCUSSION TITLE
RTMP Stream metrics

### DISCUSSION THREAD

**Discussion by jscelle** [HaishinKit/HaishinKit.swift#1565] [Q&A]
Hi, 

Is there any way to get statistics from RTMPSocket / RTMPNWSocket?
Logger does not provide any, except for chunk message, but i need to handle networking info: RTT, packet loss etc
Is there any way to do it?

**Comment by shogo4405** [comment]
The network-related statistics can be found around here.
https://github.com/shogo4405/HaishinKit.swift/blob/c961e2449d20c4824fa8c11bb5c13601c3d52652/Sources/RTMP/RTMPConnection.swift#L191-L209
